src/sentry/models/groupshare.py

```diff
@@ -16,6 +16,10 @@
 from sentry.db.models import FlexibleForeignKey, Model, BaseManager, sane_repr
 
 
+def default_uuid():
+    return uuid4().hex
```

Comment:
Should this be called `_guid`? Other functions that use `_uuid` return a binary type. While the `_guid` functions return a hex string.

Reply:
I only named the defaults based on the column name they are assigning to.